aws_lambda_powertools/logging/utils.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from typing import List, Optional, TypeVar
+
+from .logger import Logger
+
+PowertoolsLogger = TypeVar("PowertoolsLogger", bound=Logger)
+
+
+def copy_config_to_registered_loggers(
+    source_logger: PowertoolsLogger, exclude: Optional[List[str]] = None, include: Optional[List[str]] = None
```

Comment:
Missing the log level parameter

Reply:
Good catch, missed that part.Do you think we should fallback to source_logger log level when the level parameter is not provided or we should skip modifying registered logger level fully?

Reply:
> Do you think we should fall back to source_logger log level when the level parameter is not provided

Yes! That was the original behaviour :) 

```python
def copy_config_to_registered_loggers(
    source_logger: Logger,
    log_level: str = None,
    include: Optional[List[str]] = None,
    exclude: Optional[List[str]] = None,
):
    """Copies source Logger level and handler to all registered loggers for consistent formatting.

    Parameters
    ----------
    source_logger : Logger
        Powertools Logger to copy configuration from
        
    HERE
    log_level : str, optional
        Logging level to set to registered loggers, by default uses source_logger logging level
```

Reply:
modified code to reflect requirement